cmd/admin-handlers.go

```diff
@@ -71,7 +74,7 @@ func (a adminAPIHandlers) VersionHandler(w http.ResponseWriter, r *http.Request)
 	jsonBytes, err := json.Marshal(adminAPIVersionInfo)
 	if err != nil {
 		writeErrorResponseJSON(w, ErrInternalError, r.URL)
-		errorIf(err, "Failed to marshal Admin API Version to JSON.")
+		logger.LogIf(context.Background(), err)
```

Comment:
I think it would be better if we have a variable initialized at the top as `ctx := context.Background()` and `LogIf` called as `logger.LogIf(ctx, err)`. 
I believe there are also several other places where this comment can be applied.

Reply:
We will have to revisit this once this PR is merged. We might even add a `API` value for this.